### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by radlotus** [radian-software/riju#174]
Currently the website is not comfortable to use on mobile due to the body size being too big.

**Comment by raxod502** [comment]
Agreed (and that's far from the only issue). Significant design work would be needed to create a new interface for mobile, and it's unlikely that I'll undertake this, but (pending https://github.com/radian-software/riju/issues/168) I would be happy to review and merge changes to improve mobile support, assuming that they don't degrade the existing experience on desktop.

**Comment by phamduylong** [comment]
With a quick lookthrough, it looks like the front-end design is simple EJS template and Bulma UI. I'll take a shot.

**Comment by raxod502** [comment]
Before you take too much time on this, please note that it'll probably be a fair while until I'm able to merge the changes. I haven't had too much time to dedicate to active maintenance of Riju in quite a while, which means that changes elsewhere have rendered the build system and deployment pipeline no longer operable. Unfortunately, considerable architectural work is needed before I am able to deploy any changes, my own or other people's.

**Comment by phamduylong** [comment]
No problem. I do acknowledge that my changes may not align with what the maintainer of the repo thinks, if that's the case then so be it. I made a PR for this [here](https://github.com/radian-software/riju/pull/187), feel free to check it whenever fits for you.
